GITHUB101.md

```diff
@@ -0,0 +1,94 @@
+This document is aimed at getting started with github if you never used it.
+
+If you're a developer there's more info at [README.md](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/blob/master/README.md).
+
+# Quick links
+
+- [Issues assigned to Gregg](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app//issues?q=assignee%3Agreggmiller+is%3Aopen)
```

Comment:
We have actually updated the process to add a `Medic` label - can you update this to give links to medical?